### DISCUSSION TITLE
Beware of malicious clones!

### DISCUSSION THREAD

**Discussion by TheLastProject** [CatimaLoyalty/Android#1812] [General]
Catima has been gaining in popularity. While this is great, it also means it has been attracting the attention of malicious people who release unauthorized and malicious closed-source modifications of Catima. Please avoid these clones at all costs and if you find anyone using them, nudge them towards the real Catima.

# Official Catima sources

For your safety, only download Catima from one of the following sources:
- [IzzyOnDroid](https://apt.izzysoft.de/fdroid/index/apk/me.hackerchick.catima)
- [Google Play](https://play.google.com/store/apps/details?id=me.hackerchick.catima)
- [GitHub](https://github.com/CatimaLoyalty/Android/releases) (note: the GitHub build does not contain its own update checker, so you will have to manually check for updates yourself)

# Community maintained sources

- [F-Droid](https://f-droid.org/en/packages/me.hackerchick.catima/)

Community maintained sources are not published by trusted Catima team members but by third parties. While these sources are most likely safe it is important to keep in mind that the Catima team does not manage or check these distributions.

# Other sources

Versions uploaded to other websites are done by third parties and may or may not be modified to contain malware. Please avoid third party websites whenever possible. Even if you want to use an older version you do not need to use a third party source. The GitHub link contains every Catima version ever released.

# Known malicious Catima clones

**Do not download or install any of the apps in the list below. They are listed here just to raise awareness. If you see anyone use them, please tell them they are using a malicious clone and point them to Catima instead. If you can, please consider complaining to the host.**

| App name | App Store | Package ID | Malicious changes | Current status |
| --------------- | ----------------- | ----------------- | -------------------------- | ---------------------- |
| dCard | RuStore (previously Google Play) | ru.ALKmk.dCard | Added advertisements and tracking of a lot of private data including location data, removed original copyright information | Hosted on Google Play since at least since at least February 29, 2024. Google refuses to remove this illegal clone despite the supplied evidence (on July 1st, 2025, it was noticed the clone disappeared from Google Play for unknown reason). Also hosted on RuStore since at least February 8, 2024. I have not been able to find any way to report this illegal clone to them. |

# Past malicious Catima clones

These malicious clones have been removed by the app store in question.

| App name | App Store | Package ID | Malicious changes | Current status |
| --------------- | ----------------- | ----------------- | -------------------------- | ---------------------- |
| Catima (Jinlong Pan) | Samsung Galaxy Store | com.pjl.catima | Malicious clone with additional permissions, malware. | Reported to Samsung on July 5, 2024, removed within an hour of reporting |

**Comment by Espionage724** [comment]
> * [F-Droid](https://f-droid.org/en/packages/me.hackerchick.catima/)
> 
> 
> Community maintained sources are not published by trusted Catima team members but by third parties. While these sources are most likely safe it is important to keep in mind that the Catima team does not manage or check these distributions.

I've used Catima from F-Droid for a few years now, but just learned it wasn't "official" a few mins ago.

I was curious if Catima was on iOS and saw the info in DDG search:

Image:
> ![Screenshot_2024-12-12_14-33-14](https://github.com/user-attachments/assets/3779def9-39a8-4974-880d-327c6830359c)

Couldn't find any mention on https://catima.app/ and I see F-Droid front-and-center, so I figured it was an old scrape. Now I find this thread :p

I don't have any real concerns or issues with this at the moment, but just wanted to report my experience.

I see now that upon clicking F-Droid on that app page it plainly lays out the details, but I admit I only heard of Catima _from_ F-Droid and never went to the main app page until now :p

**Comment by siper** [comment]
![Screenshot_20250105-212145.png](https://github.com/user-attachments/assets/27289abd-a4ae-4cfb-8b1d-4bdf20d6fd3b)

Catima in RuStore: https://www.rustore.ru/catalog/app/me.hackerchick.catima 

I think it not original.

**Reply by TheLastProject** [reply]
That's indeed not uploaded by me.

I see a "Downloaded from Aptoide" text on the bottom of the page. However, I didn't upload it to Aptoide either. There's no way for me to check if either of these are copies uploaded by someone else or just plain malware as both RuStore and Aptoide instead download their own store when you click the download link of the app.

I would indeed strongly recommend avoiding downloading from either RuStore or Aptoide given I can't tell if this is a clean mirror or a maliciously edited version.

**Reply by siper** [reply]
I'll make a report, for RuStore administration.
